web/src/components/snapshots/Snapshots.jsx

```diff
@@ -225,18 +225,9 @@ class Snapshots extends Component {
       status: "Deleting",
       trigger: "manual",
       sequence: snapshot.sequence,
-      startedAt: Utilities.dateFormat(
-        snapshot.startedAt,
-        "MM/DD/YY @ hh:mm a z"
-      ),
-      finishedAt: Utilities.dateFormat(
-        snapshot.finishedAt,
-        "MM/DD/YY @ hh:mm a z"
-      ),
-      expiresAt: Utilities.dateFormat(
-        snapshot.expiresAt,
-        "MM/DD/YY @ hh:mm a z"
-      ),
+      startedAt: snapshot.startedAt,
```

Comment:
Are these formatted elsewhere?

Reply:
yes, so they were doubly formatted and were failing only in firefox for some reason